script/popjsonb

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+set -e
+
+beg=0
```

Comment:
Can we make it possible to set the `beg`, `inc` and `max` vars using environment variables?